CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## datadog-api-client-1.0.0-beta.3 / 2020-05-20
+
+* [Added] Add security monitoring to clients. See [#306](https://github.com/DataDog/datadog-api-client-java/pull/306).
+* [Added] Add /v1/validate endpoint. See [#297](https://github.com/DataDog/datadog-api-client-java/pull/297).
+* [Added] Add a .generated_files. See [#274](https://github.com/DataDog/datadog-api-client-java/pull/274).
```

Comment:
This doesn't impact functionality, but its an addition to the repo, so I'm not sure it needs a callout, so I left this.

Reply:
I don't see a problem with keeping this changelog line.